Fixed extension for Rerun and Junit reports

JasonSmiley208 opened the pull request:
Should use .txt for rerun reports (not .rerun). Should use .xml for junit reports (not .junit). Also I updated the report names to use the correlating generated IT file.

JasonSmiley208 commented on src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java:
```diff
@@ -193,9 +193,18 @@ private String createFormatStrings() {
             sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
                     config.getCucumberOutputDir()
-                    .replace('\\', '/'), fileCounter, formatStr));
+                    .replace('\\', '/'), outputFileName, extension));
```
using the outputFileName instead of fileCounter since I think it makes sense to make the output files match the associated IT Generated file.

I would think this just makes it easier to determine the associated IT file.

temyers commented:
There are a number of integration test failures.

Please run `mvn clean install -Prun-its` and fix the test failures.

JasonSmiley208 commented:
I'll fix the tests. Thanks for fixing the CI :)

JasonSmiley208 commented:
I've been developing on github... not sure how to edit two different files :(

JasonSmiley208 commented:
So I just removed the File Name update - I was having issues working on my fork and rather trying to figure that out, I just removed the test breaking change. This pull request now just updated the extension

temyers commented:
Since this is part of #30, this should be a PR onto @sugatmankar repository.

Please update and open a PR onto https://github.com/sugatmankar/cucumber-jvm-parallel-plugin instead
